contrib/kml_export.lua

```diff
@@ -230,23 +229,16 @@ function addDuplicateIndex( index, filename )
 end
 
 local function create_kml_file(storage, image_table, extra_data)
-  if not df.check_if_bin_exists("mkdir") then
-    dt.print_error(_("mkdir not found"))
-    return
-  end
   if not df.check_if_bin_exists("convert") then
     dt.print_error(_("convert not found"))
     return
   end
-  if not df.check_if_bin_exists("xdg-open") then
-    dt.print_error(_("xdg-open not found"))
-    return
-  end
-  if not df.check_if_bin_exists("xdg-user-dir") then
-    dt.print_error(_("xdg-user-dir not found"))
-    return
+  if dt.configuration.running_os == "windows" then
+    if not df.check_if_bin_exists("xdg-user-dir") then
```

Comment:
Similar to my comment above, my experience has been this will fail unless you provide the user a way to specifically define a directory for this and at some point do a dt.preferences.write.

Reply:
this was stupid, the right syntax must be 
if dt.configuration.running_os == "linux" then